nifi-nar-bundles/nifi-provenance-repository-bundle/nifi-persistent-provenance-repository/src/main/java/org/apache/nifi/provenance/lucene/DocsReader.java

```diff
@@ -47,9 +46,6 @@
 public class DocsReader {
     private final Logger logger = LoggerFactory.getLogger(DocsReader.class);
 
-    public DocsReader(final List<File> storageDirectories) {
-    }
-
```

Comment:
I'm like 95% with @olegz  on this. I believe it is implausible that someone would be calling this method and likely needed a comment that it was public for some other reason than it was a public API. That 5% not with @olegz is adding deprecation tag adds a tiny smidge of tech debt, which would be swept up when we investigate all the deprecated methods for 1.0.0